lolautruche/payline-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "bundles": {
+        "Lolautruche\\PaylineBundle\\LolautruchePaylineBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "PAYLINE_MERCHANT_ID": "your_payline_merchant_id",
+        "PAYLINE_ACCESS_KEY": "your_payline_access_key",
```

Comment:
New practice here: I think env var for secrets should be commented, in order to allow managing secrets with the new `secrets:*` commands
`"#PAYLINE_ACCESS_KEY": "your_payline_access_key",`

Reply:
Won't this make it unavailable then? If so, that would generate an exception as these information are mandatory

Reply:
One will get an exception when accessing the service while the env var is not set - but a random string is not better, wdyt? At compile time, this should make no difference as no env vars should be read then.
Can you give it a try?